tide/master/rest/RestInterface.h

```diff
@@ -71,6 +73,8 @@ class RestInterface : public QObject
     /** Out-of-line destructor. */
     ~RestInterface();
 
+    void setLogger(LoggingUtility& logger);
```

Comment:
I think this could be a:
const LoggingUtility& logger
everywhere (also in RestLogger)